.rubocop.yml

```diff
@@ -29,8 +34,12 @@ AllCops:
     - 'fastlane/Fastfile'
   Exclude:
     - 'lib/assets/custom_action_template.rb'
+    - 'lib/fastlane/actions/xcodebuild.rb'
 
-
+Rails/TimeZone:
```

Comment:
Please remove from here. If you want to keep it, add it to the `xcodebuild.rb` file instead.